transport/x/grpc/codec.go

```diff
@@ -47,9 +47,11 @@ func (customCodec) Unmarshal(data []byte, obj interface{}) error {
 }
 
 func (customCodec) String() string {
-	// TODO: faking this as proto to be compatible with existing grpc clients
-	// https://github.com/yarpc/yarpc-go/issues/911
-	return "proto"
+	// setting this to what amounts to a nonsense value
```

Comment:
This isn't a complete sentence, we have comments like this all over the codebase (and appropriately so)